tmcprototype/DishLeafNode/DishLeafNode/DishLeafNode.py

```diff
@@ -297,6 +432,10 @@ def convert_radec_to_azel(self, data):
         doc="FQDN of Dish Master Device",
     )
 
+    TrackDuration = device_property(
```

Comment:
`TrackDuration` will be specified per command. Hence, it is better implemented as an input argument (`argin`) of Track instead of  DeviceProperty.

Reply:
Track Duration as an input argument will addressed in the coming PI.

 TrackDuration is added to devices.json file
Commit ID : 2e978037512d9ba47e7b5d8f8bf016ce6633d4dd